core/templates/dev/head/galleries/Gallery.js

```diff
@@ -275,6 +284,7 @@ oppia.controller('SearchBar', [
       summary: ''
     }
   };
+  $scope.translationData = {}
```

Comment:
This was a hard one, the original name was taken from the angular-translate documentation. This variable is a map between elements of the screen and non translatable portions of html strings. I can't put all that in the variable name! I added a comment and change the variable for nonTranslatableStrings, but probably someone can come up with a better name :D